src/app/src/util/propTypes.js

```diff
@@ -179,6 +179,27 @@ export const countryOptionsPropType = arrayOf(
     }),
 );
 
+export const facilityTypeOptionsPropType = arrayOf(
+    shape({
+        value: string.isRequired,
+        label: string.isRequired,
+    }),
+);
+
+export const processingTypeOptionsPropType = arrayOf(
+    shape({
+        value: string.isRequired,
+        label: string.isRequired,
+    }),
+);
+
+export const facilityProcessingTypeOptionsPropType = arrayOf(
```

Comment:
Basically the same structure are the response from the endpoint since we parse the types in react